proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/saml/HubResponseTranslator.java

```diff
@@ -44,7 +44,7 @@ public HubResponseTranslator(
         this.proxyNodeMetadataForConnectorNodeUrl = proxyNodeMetadataForConnectorNodeUrl;
     }
 
-    Response translate(HubResponseContainer hubResponseContainer) {
+    public Response translate(HubResponseContainer hubResponseContainer) {
```

Comment:
Why make this method public? It's only accessed in its package hence the default access level of package-private (no modifier)